src/debugpy/_vendored/pydevd/pydevd_attach_to_process/attach_pydevd.py

```diff
@@ -38,6 +38,7 @@ def process_command_line(argv):
 
 
 def main(setup):
+    sys.path.append(os.path.dirname(__file__))
```

Comment:
This is actually unrelated (it's an issue in the attach to process where under a particular scenario the `add_code_to_python_process` couldn't be imported).